src/components/Match/StyledMatch.jsx

```diff
@@ -233,6 +233,17 @@ export const StyledStoryWrapper = styled.div`
 export const StyledStoryNetWorthBar = styled.div`
 
   display: flex;
+  position: relative;
+
+  &::after {
+    content: "";
+    position: absolute;
+    width: 4px;
+    height: 13px;
+    background-color: white;
+    left: 50%;
+    margin-left: -2px;
```

Comment:
what's this for?

Reply:
moves it 2px to the left so it's precisely centered

Reply:
Oh I see it's half of the width value. I generally dislike CSS that's bound to itself like this because it's easy to accidentally break. I don't have a better solution though aside from styling this in JS.